internal/git/commands_test.go

```diff
@@ -447,14 +940,12 @@ func TestBackendCommands(t *testing.T) {
 			Branch:      "branch",
 			FileContent: "branch content",
 			FileName:    "file",
-			Locations:   testgit.Locations{testgit.LocationLocal},
```

Comment:
Could you clarify the motivation for removing this line? It seems unrelated to the other changes in this pull request.

Reply:
It didn't seem necessary and the tests still passed without it, so I thought to clean it up. That should probably happen in a different PR, though, so the change can be discussed on its own. I'll revert it here.

Reply:
Yes, I have [disabled](https://github.com/git-town/git-town/blob/main/Makefile#L93) the [exhaustruct](https://github.com/GaijinEntertainment/go-exhaustruct) linter for test code a while ago, to allow tests to be more concise.

There are probably a whole number of struct fields in tests that could be deleted as a result. I agree that's a whole other can of worms that is better done in a separate PR.